language.py

```diff
@@ -1110,6 +1110,30 @@ def getTypeConstraints(self, bitwidth):
     return And(self.type == self.val.type, self.type.getTypeConstraints(bitwidth))
 
 
+def to_bitwidth(obj):
+    t = str(obj.type)
```

Comment:
@goens could you change this to checking `isinstance(obj.type, IntegerType)` and `isinstance(obj.type, UnknownType)`?